models/userDao.js

```diff
@@ -0,0 +1,42 @@
+const appDataSource = require("./appDataSource");
+
+const createUser = async (email, name, password, address, phone, birthdate) => {
+  try {
+    return await myDataSource.query(
+      `INSERT INTO users(
+        name,
+        password,
+        address,
+        phone,
+        email,
+        birthdate
+        ) VALUES (?, ?, ?, ?, ?, ?);
+  `,
+      [name, password, address, phone, email, birthdate]
+    );
+  } catch (err) {
```

Comment:
```suggestion
  } catch {
```
try 블럭에서 발생하는 특정 예외(err이라는 이름의)가 아닌 모든 exeption에 대해 catch가 잡고 있기 때문에 위와같이 작성되어도 의도하신 바와 똑같이 동작할 것입니다!

Reply:
네 꿀팁감사합니다 !